.drone.yml

```diff
@@ -0,0 +1,48 @@
+kind: pipeline
```

Comment:
wow, good job! Did you use any other `.drone.yml` as a base for this? Which one?

Reply:
yep! I used the one I created for [fastcheckout](https://github.com/vtex/fastcheckout/blob/main/.drone.yml)